### PR TITLE
OOP school library: set up associations

### PR DESCRIPTION
**Pull request by cwaku**
## In this PR:
I added associations to the project's objects as required by the project requirements

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :label, :author, :title
+  attr_reader :rentals
```
Comment (enaburekhan) [inline book.rb]: - [x] The `:rentals` variable should have an `attr_accessor`. It will be nice to fix this.

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```
Comment (enaburekhan) [inline rental.rb]: - [x] Bothe `:book` and `:person` should have an `attr_accessor`. It will be nice to fix this.

File: rental.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  def initalize(date, person, book)
+    @date = date
+    @person = person
+    person.rentals << self
+    @book = book
+    book.rental << self
```
Comment (enaburekhan) [inline rental.rb]: - [x] I will suggest you fix this syntax error: `book.rental` to `book.rentals`

File: person.rb [hunk 4]
```diff
@@ -1,68 +1,38 @@
...
-  end
-end
+require './validate'
+require './rental'
 
-class Person < Namable
+class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```
Comment (enaburekhan) [inline person.rb]: - [x] `:rentals` should be part of the `attr_accessor`. It will be nice to fix this.

**Review by enaburekhan — CHANGES_REQUESTED** [review]
Hi cwaku, 

Good job so far!

- [ ] Major highlights

- The PR commit messages are very descriptive
- The readme is also very descriptive.
- Passed linters checks.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Ricky, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**